File: Program/ActEditor/src/pl/wroc/pwr/student/acteditor/parsing/xml/XMLFormatter.java
package pl.wroc.pwr.student.acteditor.parsing.xml;

import java.util.ArrayList;
import java.util.List;

import pl.wroc.pwr.student.acteditor.parsing.Formatter;

/**
 * Zapewnia formatowanie danych do dokumentu XML.
 * 
 * @author Mateusz
 *
 */
public class XMLFormatter implements Formatter {
	private List tags = new ArrayList();

	@Override
	public void add(String tag) {
		tags.add(tag);
	}

	@Override
	public String generate() {
		return null;
	}
}

File: Program/ActEditor/bin/pl/wroc/pwr/student/acteditor/parsing/xml/XMLFormatter.js
Clazz.declarePackage ("pl.wroc.pwr.student.acteditor.parsing.xml");
Clazz.load (["pl.wroc.pwr.student.acteditor.parsing.Formatter", "java.util.ArrayList"], "pl.wroc.pwr.student.acteditor.parsing.xml.XMLFormatter", null, function () {
c$ = Clazz.decorateAsClass (function () {
this.tags = null;
Clazz.instantialize (this, arguments);
}, pl.wroc.pwr.student.acteditor.parsing.xml, "XMLFormatter", null, pl.wroc.pwr.student.acteditor.parsing.Formatter);
Clazz.prepareFields (c$, function () {
this.tags =  new java.util.ArrayList ();
});
Clazz.overrideMethod (c$, "add", 
function (tag) {
this.tags.add (tag);
}, "~S");
Clazz.overrideMethod (c$, "generate", 
function () {
return null;
});
});
